packages/forklift-console-plugin/src/mock-console-extension/setupBrowserWorker.tsx

```diff
@@ -1,27 +1,31 @@
-import { type SetupWorker, rest, setupWorker } from 'msw';
+import { rest, SetupWorker, setupWorker } from 'msw';
 
-import { allHandlers } from '@kubev2v/mocks/consoleHandlers';
+import { getMockDataForRequest } from '@kubev2v/mocks';
 
-export { type SetupWorker } from 'msw';
+import { logMockRequest } from './logMockRequest';
 
 /**
  * Setup the MockServiceWorker with all of the mock handlers available handling request
  * for UI url paths.  The worker will be added to `window.msw` for debugging convenience.
  *
- * @param staticFilePath URL prefix where `mockServiceWorker.js` will be available
+ * @param staticFilePath - URL prefix where `mockServiceWorker.js` will be available
  * @returns A setup and started MockServiceWorker
  */
 export function setupBrowserWorker(staticFilePath: string): SetupWorker {
   const worker = setupWorker(
-    ...allHandlers,
-
-    rest.all('/*', (req) => {
-      console.log(
-        '%cmock-plugin passthrough%c \u{1fa83} %s',
-        'font-weight: bold',
-        'font-weight: normal',
-        req.url,
-      );
+    rest.all('/*', (req, res, ctx) => {
```

Comment:
@sjd78 
1. is it OK to redefine the path reg-exp?  the `all` handler is defined as `/.+/` by the library - https://github.com/mswjs/msw/blob/85e4f6ff98b3c46b30ae0f62c962489205992d62/src/rest.ts#L35 
2.  can the same request be handled by multiple handlers? before we had `allHandlers` and then `all` handler